### ISSUE TITLE
This installation script can not be ran in WSL2

### ISSUE THREAD

**Issue by ethever** [nevillegrech/gigahorse-toolchain#160]
This installation script can not be ran in WSL2
```bash
curl -s -L https://raw.githubusercontent.com/nevillegrech/gigahorse-toolchain/master/scripts/docker/install/install_amd64 | bash
```
because the ran results of this command concated the value instead of the refrence of the $PATH environment variable.

**Comment by ethever** [comment]
This pr could fix it: #161

**Comment by sifislag** [comment]
Closed by merging #161
